Client/Frontend/Library/Bookmarks/BookmarkDetailPanel.swift

```diff
@@ -415,16 +415,24 @@ class BookmarkDetailPanel: SiteTableViewController {
 
         switch indexPath.row {
         case BookmarkDetailFieldsRow.title.rawValue:
-            cell.titleLabel.text = .BookmarkDetailFieldTitle
-            cell.textField.text = bookmarkItemOrFolderTitle
-            cell.textField.autocapitalizationType = .sentences
-            cell.textField.keyboardType = .default
+            if let bookmarkTitle = bookmarkItemOrFolderTitle {
```

Comment:
I think it would be prefereable to pass an optional on `textFieldText` for both cases here, so we still configure the title and other properties in case the `textFieldText` is not there.